Functions/BIDS/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -151,8 +161,16 @@
 		if dimASL(4)>=numel(jsonOut.GELabelingDuration)
 			warning('Labeling duration mismatch with GE private field.');
 			jsonOut.LabelingDuration = jsonOut.GELabelingDuration;
+		elseif dimASL(4)>=numel(unique(jsonOut.GELabelingDuration))
+			warning('Labeling duration mismatch with GE private field.');
```

Comment:
Which private field? Add the name/number to the warning. Helps us debugging.

Reply:
No point adding DICOM tag here - cause that is read elsewhere or in dcm2nii, so that would be prone to future error.